### ISSUE TITLE
How to pass parameter(arguments) to the function while solving the optimization?

### ISSUE THREAD

**Issue by sathishlsk** [CMA-ES/pycma#128]
I am trying to pass the arguments to the function which I am optimization. but I couldn't able to pass the arguments to the function. Can you please help me out with this. Thanks in advance.

**Comment by nikohansen** [comment]
You provide very little information to go by. Maybe check out
```python
import cma
help(cma.fmin)
help(cma.interfaces.OOOptimizer.optimize)
```
 and look for `args`.

**Comment by sathishlsk** [comment]
Thanks. Let me check it

**Comment by ndsanders** [comment]
Hi, Niko,

I think I'm having a similar issue along this lines in the latest version (3.0.2). The example below demonstrates my problem.

```python3
import cma
import numpy as np

def f(x, a):
  return a * np.sum(x)

cma.CMAEvolutionStrategy([0, 0], 0.5).optimize(f, args=[4.0])
```

This would have worked in versions < 3.0.1, but now it produces the following error
```
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/Users/nicksanders/anaconda3/lib/python3.6/site-packages/cma/interfaces.py", line 208, in optimize
    #fitvals = eval_all(X, *args)
  File "/Users/nicksanders/anaconda3/lib/python3.6/site-packages/cma/optimization_tools.py", line 268, in __call__
    return [fitness_function(x, *args) for x in solutions]
  File "/Users/nicksanders/anaconda3/lib/python3.6/site-packages/cma/optimization_tools.py", line 268, in <listcomp>
    return [fitness_function(x, *args) for x in solutions]
TypeError: 'float' object is not callable
```

The issue seems to come from the recent switch to`EvalParallel` and `EvalParallel2`. If I un-comment the old code (line 207 of `cma/interfaces.py`) then I can get it working as usual. I'm unable to see why `EvalParallel` is giving me this problem.

**Comment by nikohansen** [comment]
That happens when not getting enough sleep and having poor test coverage.

Like in anticipation, it was a nice trick though to leave the old code commented :-)

Line 208 of interfaces.py must read
```python
    fitvals = eval_all(X, args=args)
```
Thanks for the catch! Will be fixed in `3.0.3`.

**Comment by nikohansen** [comment]
Fixed with 121215a35e145170cbad7a46789e603598bca956 and released on [PyPI](https://pypi.org/project/cma/).
